core/src/main/java/org/apache/carbondata/core/scan/result/vector/impl/CarbonColumnVectorImpl.java

```diff
@@ -102,6 +126,58 @@ public CarbonColumnVectorImpl(int batchSize, DataType dataType) {
 
   }
 
+  @Override
+  public CarbonColumnVector getColumnVector() {
+    return null;
+  }
+
+  @Override
+  public List<CarbonColumnVectorImpl> getChildrenVector() {
+    return childrenVector;
+  }
+
+  @Override
+  public void putArrayObject() {
+    return;
+  }
+
+  public void setChildrenVector(ArrayList<CarbonColumnVectorImpl> childrenVector) {
+    this.childrenVector = childrenVector;
+  }
+
+  public ArrayList<Integer> getChildrenElements() {
+    return childrenElements;
+  }
+
+  public void setChildrenElements(ArrayList<Integer> childrenElements) {
+    this.childrenElements = childrenElements;
+  }
+
+  public ArrayList<Integer> getChildrenOffset() {
+    return childrenOffset;
+  }
+
+  public void setChildrenOffset(ArrayList<Integer> childrenOffset) {
+    this.childrenOffset = childrenOffset;
+  }
+
+  public void setChildrenElementsAndOffset(byte[] childPageData) {
+    ByteBuffer childInfoBuffer = ByteBuffer.wrap(childPageData);
+    ArrayList<Integer> childElements = new ArrayList<>();
+    ArrayList<Integer> childOffset = new ArrayList<>();
```

Comment:
offset is not required, even for struct type. so, please remove it

Reply:
okay, removed